_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 'Bitcoin Optech Newsletter #107'
+permalink: /en/newsletters/2020/07/22/
+name: 2020-07-22-newsletter
+slug: 2020-07-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to several discussions about activating
+taproot and summarizes a proposed update to BIP173 bech32 addresses.
+Also included are our regular sections summarizing interesting changes to
+services and client software, releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Taproot activation discussions:** several discussions were started
+  or continued this week about choosing a method for activating
+  [taproot][topic taproot].
+
+    - **New IRC room:** Steve Lee [posted][lee irc] to the Bitcoin-Dev
+      mailing list an announcement of a new `##taproot-activation` IRC
+      room on the Freenode.net network.  The channel is [logged][irc
+      log] and saw an impressive amount of conversation from dozens of
+      participants during its first week.
+
+    - **Mailing list thread:** Anthony Towns posted a summary of
+      the recently-updated [BIP8][] activation method (see [Newsletter
+      #104][news104 bip8]) and of Towns's own new [bip-decthresh][]
+      activation proposal (based on Matt Corallo's "modern soft fork
+      activation" [post][msfa] in January, see [Newsletter #80][news80
+      activation]).  An interesting feature of the decreasing threshold
+      idea is that the amount of network hashrate needed to signal
+      readiness to enforce the new soft fork rules would decrease during
+      the final activation period, allowing miner activation with
+      perhaps as little as 60% miner support rather than the 95% miner
+      support needed during the first activation period.
+
+- **Bech32 address updates:** Russell O'Connor [posted][oconnor post] to
+  the Bitcoin-Dev mailing list a reply to Pieter Wuille's November post
+  (see [Newsletter #77][news77 bech32]) about amending BIP173 to only
+  allow witness programs of either 20 or 32 bytes, minimizing the risk
+  that typos would cause a user to irrecoverably send bitcoins to an
+  wrong address.  O'Connor made the counter-proposal that we could allow
```

Comment:
```suggestion
  that typos would cause a user to irrecoverably send bitcoins to a
  wrong address.  O'Connor made the counter-proposal that we could allow
```